doc/connectivity/bluetooth/bluetooth-shell.rst

```diff
@@ -0,0 +1,357 @@
+.. _bluetooth_shell:
+
+Bluetooth Shell
+###############
+
+Bluetooth Shell Setup and Usage
+*******************************
+
+First you need to build and flash your board with the Bluetooth shell. For more details on that,
+look at the :ref:`getting_started`. The Bluetooth shell itself is located in
+:zephyr_file:`tests/bluetooth/shell/`.
+
+When it's done, connect to the CLI using your favorite application. You should be welcomed by the
+following prompt:
+
+.. code-block:: console
+
+        uart:~$
+
+For more details on the common usage of the shell, see :ref:`shell_api`.
+
+The first thing you need to do is enable Bluetooth. To do, use :code:`bt init`. Messages should
+confirm you the initialization with a bunch of logging message, if enabled.
+
+Identities
+**********
+
+The shell allows the creation of multiple identities, the maximum being set by the Kconfig symbol
+:kconfig:option:`CONFIG_BT_ID_MAX`. To create a new identity, use :code:`bt id-create`. You can then
+use it by selecting it with its ID :code:`bt id-select <id>`. Finally, you can list all the
+identities available with :code:`id-show`.
```

Comment:
`available identities.`

Could you add another sentence briefly describing the concept of identity? Since we don't really have it anywhere; we will move it when we add stack documentation for it.

Something like:
`Identities are a Zephyr host concept, allowing a single physical device to behave like multiple logical Bluetooth devices.`